test/tests/dialog-modal_datepicker.js

```diff
@@ -801,34 +838,30 @@ ariaTest(
   exampleFile,
   'grid-shift-pageup',
   async (t) => {
-    await t.context.session.findElement(By.css(ex.buttonSelector)).click();
-    let day = new Date();
+    const startDate = '1/31/2023';
+    const targetDates = [
+      '1/31/2022',
+      '1/31/2021',
+      '1/31/2020',
+      '1/31/2019',
+      '1/31/2018',
+      '1/31/2017',
+    ];
```

Comment:
This doesn't demonstrate the bug fix (it passes on the `main` branch). Starting from 2020-02-29 would do it, though

```suggestion
    const startDate = '2/29/2020';
    const targetDates = [
      '2/28/2019',
      '2/28/2018',
      '2/28/2017',
      '2/29/2016',
      '2/28/2015',
      '2/28/2014',
    ];
```

Reply:
@jugglinmike 
Thank you for the feedback, I updated the code.